.github/workflows/_deploy-github-pages.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+# This workflow deploys the documentations to GitHub Pages.
+name: Deploy static content to Pages
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        type: string
+        required: true
+        description: The artifact name that contains docs content
+      artifact-sha256:
+        type: string
+        required: true
+        description: The sha of the artifact that contains docs content
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
```

Comment:
From my understanding, at any time, there can only be at most one in-progress run and at most one pending run for this workflow.
Trigger this workflow will removed any pending run. If there is an in-progress run, it will put itself to pending. If there is NO in-progress run, it will run. :thinking:

Reply:
Sorry I don't understand your question.

Reply:
Oh I was trying to confirm if my understanding for the usage of `concurrency` here is correct.

Reply:
Otherwise, everything LGTM.

Reply:
> Oh I was trying to confirm if my understanding for the usage of `concurrency` here is correct.

Yes, you're correct. Thinking some more I decided to remove the `concurrency` condition as it's very unlikely to happen for us, and it's better to be consistent with other workflows.

See commit [43765be](https://github.com/oracle-samples/macaron/pull/249/commits/43765be9dea9ddf5f42f6e5c7f77b76e6a007c58).